cinema/views.py

```diff
@@ -69,7 +75,7 @@ class MovieViewSet(
 ):
     queryset = Movie.objects.prefetch_related("genres", "actors")
     serializer_class = MovieSerializer
-    authentication_classes = (TokenAuthentication,)
+
```

Comment:
The `permission_classes` attribute is correctly set to `IsAdminOrIfAuthenticatedReadOnly`, which aligns with the intended access control for this viewset.